src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -779,13 +781,13 @@ private async Task UpdateParametersAndCompleteJob()
                 }
             }
 
-            _logger.LogTrace("Completing reindex job. Updating the status of the fully indexed search parameters: '{ParamUris}'", string.Join("', '", fullyIndexedParamUris));
+            _logger.LogTrace("Reindex job updating the status of the fully indexed search, id: {Id}, parameters: '{ParamUris}'", _reindexJobRecord.Id, string.Join("', '", fullyIndexedParamUris));
```

Comment:
Nit: Should this be a LogInformation to be consistant?

Reply:
Yeah, I can update it to that.